public/plugin.ts

```diff
@@ -155,7 +155,11 @@ export class SecurityPlugin
       })
     );
 
-    if (multitenancyEnabled && config.multitenancy.enable_aggregation_view) {
+    if (
+      multitenancyEnabled &&
+      config.multitenancy.enabled &&
```

Comment:
Just wanna double check L159 here. So is that mean that the configuration in frontend is more like a feature flag for dynamical configurable multi-tenancy feature?